tslint.json

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
You don't have to configure all that manually, it would be endless to do so ;) tslint has some full default config from which you can inherit and eventually disable some not-so-important rules.

Check how I use the `extends` [there](https://github.com/Electra-project/Electra-JS/blob/master/tslint.json).

Reply:
done